doc/en/exporters.rst

```diff
@@ -244,6 +244,18 @@ A more explicit usage is to initialize a JSON exporter directly:
     def main(plan):
         ...
 
+With argument ``--split-json`` the generated JSON report can be split into
```

Comment:
as discussed in team, we only simply add a cfg option `split_json_report` into `JSONExporter`, it won't bother the OSS users outside, generally users would prefer a single json file over one file with 2 attachments, much easier to store. The feature of splitting JSON is implemented for us because we use CouchDB and would like to raise the db performance. So we would like to place the code in internal repo and in the future when UI is ready we could change the default option of `split_json_report` to True.